### PR TITLE
update the description for includeCustomCacheKeyInSubject

### PR DESCRIPTION
**Pull request by utle**
includeCustomCacheKeyInSubject - add more description

### CODE REVIEW

**Comment by utle** [comment]
#build

File: dev/com.ibm.ws.security.spnego/resources/OSGI-INF/l10n/metatype.properties [hunk 1]
```diff
@@ -59,7 +59,7 @@ includeClientGSSCredentialInSubject=Add the client delegation credentials to sub
 includeClientGSSCredentialInSubject.desc=Specifies whether the client delegation credentials should be stored in a client subject.
 
 includeCustomCacheKeyInSubject=Add the custom cache key to subject
-includeCustomCacheKeyInSubject.desc=Specifies whether the custom cache key should be stored in a client subject and LTPA cookie.
+includeCustomCacheKeyInSubject.desc=Specifies whether the custom cache key should be stored in a client subject and LTPA cookie. If this property is set to true and the cache key for a user is not found in the authentication cache, the user will be required to log in again. Set this property to false when you use the SPNEGO feature to allow the security subject to be constructed from the LTPA cookie and the user registry. If the security subject is reconstructed from the user registry, there will be no Kerberos and GSS Credentials in the subject. If your LTPA cookie is used by more than one server, consider setting this property to false.
```
Comment (dmuelle) [inline dev/com.ibm.ws.security.spnego/resources/OSGI-INF/l10n/metatype.properties]: suggested edit for ID guidelines:

Specifies whether the custom cache key is stored in a client subject and LTPA cookie. If this property is set to true and the cache key for a user is not found in the authentication cache, the user is required to log in again. Set this property to false when you use the SPNEGO feature to allow the security subject to be constructed from the LTPA cookie and the user registry. If the security subject is reconstructed from the user registry, no Kerberos or GSS credentials are included in the subject. If more than one server uses your LTPA cookie , consider setting this property to false.

**Review by dmuelle — CHANGES_REQUESTED** [review]
see suggested edit, let me know if you have any questions. Thanks

File: dev/com.ibm.ws.security.spnego/resources/OSGI-INF/l10n/metatype.properties (hunk 1 above)
Comment (utle) [reply dev/com.ibm.ws.security.spnego/resources/OSGI-INF/l10n/metatype.properties]: Done

**Comment by utle** [comment]
#build

File: dev/com.ibm.ws.security.spnego/resources/OSGI-INF/l10n/metatype.properties (hunk 1 above)
Comment (dmuelle) [reply dev/com.ibm.ws.security.spnego/resources/OSGI-INF/l10n/metatype.properties]: For some reason, I dont see any changes

**Comment by utle** [comment]
#build

File: dev/com.ibm.ws.security.spnego/resources/OSGI-INF/l10n/metatype.properties (hunk 1 above)
Comment (utle) [reply dev/com.ibm.ws.security.spnego/resources/OSGI-INF/l10n/metatype.properties]: forgot to push it. Done now

**Review by dmuelle — APPROVED** [review]
LGTM

**Comment by utle** [comment]
#run-libby-bot

**Pull request merged**